### PR TITLE
feat: update to specify topic types and add msg types for tests

### PR DESCRIPTION
**Pull request by inuzukam**
## What?
- Enable to test some msg types

## Why?
- to test some msg types

### CODE REVIEW

File: pydtk/models/rosbag2.py [hunk 1]
```diff
@@ -136,7 +136,7 @@ def _load(
                 if timestamp == timestamps[timestamp_idx]:
                     msg_type = get_message(type_map[topic_])
                     msg = deserialize_message(data_, msg_type)
-                    data.append(msg.data)
+                    data.append(msg)
```
Comment (kan-bayashi) [inline pydtk/models/rosbag2.py:139]: Please fix all `msg.data` to `msg`.

Comment (inuzukam) [reply pydtk/models/rosbag2.py:139]: fixed

File: tox.ini [hunk 2]
```diff
@@ -22,6 +22,13 @@ passenv = PYTHONPATH
 commands =
     pytest -m "ros" test/
 
+[testenv:py3-pytest-ros2]
+whitelist_externals = pytest
+skip_install = true
+passenv = PYTHONPATH
```
Comment (kan-bayashi) [inline tox.ini:28]:
```suggestion
passenv = PYTHONPATH AMENT_PREFIX_PATH
```

**Review by kan-bayashi — APPROVED** [review]
LGTM

**Pull request merged**